assets/src/state.ts

```diff
@@ -29,6 +30,7 @@ export interface State {
   selectedShuttleRunIds: RunId[] | "all"
   selectedVehicleId?: VehicleId
   settings: Settings
+  vehicleAndRouteForNotification?: VehicleOrGhostAndRoute
```

Comment:
We've been using `state.ts` just for interface stuff, and the data lives in the state of the hook that gets it. So this would be like `selectedTripIdsForNotification`, and somewhere you would have `const vehicleAndRoute = useVehicleAndRouteForNotification(selectedTripIdsForNotification)`, where there's a `setState` or `useChannel` to save the data inside that hook.